extensions/amp-ad/0.1/amp-ad-3p-impl.js

```diff
@@ -498,6 +454,9 @@ export class AmpAd3PImpl extends AMP.BaseElement {
       this.xOriginIframeHandler_.freeXOriginIframe();
       this.xOriginIframeHandler_ = null;
     }
+    if (this.uiHandler) {
+      this.uiHandler.cleanup();
```

Comment:
do we need to set to null?

Reply:
I do not know if there are any adverse consequences to this, so I'd keep it the way it is.